core/vm-common/hookCrypto.go

```diff
@@ -0,0 +1,17 @@
+package vmcommon
+
+// CryptoHook interface for VM krypto functions
+type CryptoHook interface {
```

Comment:
Please move to interface.go and add IsInterfaceNil method

Reply:
done